infogami/plugins/wikitemplates/code.py

```diff
@@ -4,13 +4,17 @@
 from __future__ import print_function
 
 import os
+import sys
 try:
     from collections.abc import Mapping
 except ImportError:
     from collections import Mapping
 
 import web
-
+if "pytest" in sys.modules:
```

Comment:
Can we have a comment on this line? I'm not too sure what it's doing. If this is overriding something for testing purposes, it would be great if we could keep it in the test files, but it's ok if that's not possible :)

Reply:
Removed these changes. Will attack in a separate PR.